.github/workflows/build-and-push.yml

```diff
@@ -0,0 +1,32 @@
+# Only run on merges to trunk
+name: Build and Push Image
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build_and_push:
+    runs-on: ubuntu-latest
+  
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set vars
+        run: |
+          DOCKER_TAG=$(date '+%Y%m%d.%H%M%S')
+          echo "DOCKER_TAG=$DOCKER_TAG" >> $GITHUB_ENV
+      
+      - name: Log in to Docker Hub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+      
+      - name: Build and push
+        uses: docker/build-push-action@v2
+        with:
+          context: ./resource-synchronizer
```

Comment:
Code changes to it are probably going to be rare, but for completeness should we also push up `openstax/sync-resource` from `./resource-trigger` in this workflow?

Reply:
Oh I didn't realize there was another image that needed to be pushed. I can add that.